src/format/template.md

```diff
@@ -0,0 +1,7 @@
+{{ tag }}
+
+## {{ title }}
+
+{{ body }}
+
+<p align="right">Report generated by <a href="https://github.com/ArtiomTr/jest-coverage-report-action">🧪jest coverage report action</a> from {{ sha }}</p>
```

Comment:
update to step-security from artiom. please search all the places that need replacement

Reply:
done